test/unit-tests/controller.test.ts

```diff
@@ -2707,11 +2839,19 @@ contract(
             data: ZERO_ADDR,
           },
         ]
-        await usdc.approve(marginPool.address, collateralToDeposit, { from: accountOwner1 })
+        await borrowableMarginPool.setOTokenBuyerWhitelistedStatus(holder1, true, { from: owner })
+        finalMarginPool =
```

Comment:
We should test this flow, but it should be in its own file vs removing test functionality that exists (add vs change tests).

Reply:
I added a `controllerBorrowableMarginPoo.test.ts` in addition to the`controller.test.ts`. 

In `controllerBorrowableMarginPoo.test.ts` I test the changed flow.
In `controller.test.ts` I test the flow from this comment as well as from comment above (non borrowable accountOwner1)